contributors/guide/style-guide.md

```diff
@@ -0,0 +1,658 @@
+---
+title: Documentation Style Guide
+---
+
+# Documentation Style Guide
+
+This style guide is for content in the Kubernetes github [community repository].
+It is an extension of the [Kubernetes documentation style-guide].
+
+These are **guidelines**, not rules. Use your best judgement.
+
+* [Cheatsheet](#cheatsheet)
+* [Content design, formatting, and language](#content-formatting-and-language)
+  * [Contact information](#contact-information)
+  * [Dates and times](#dates-and-times)
+  * [Diagrams, images and other assets](#diagrams-images-and-other-assets)
+  * [Document Layout](#document-layout)
+  * [Formatting text](#formatting-text)
+  * [Language, grammar, and tone](#language-grammar-and-tone)
+  * [Moving a document](#moving-a-document)
+  * [Punctuation](#punctuation)
+  * [Quotation](#quotation)
+* [Markdown formatting](#markdown-and-formatting)
+  * [Code Blocks](code-blocks)
+  * [Emphasis](#emphasis)
+  * [Headings](#headings)
+  * [Horizontal Lines](#horizontal-lines)
+  * [Line Length](#line-length)
+  * [Links](#links)
+  * [Lists](#lists)
+  * [Metadata](#metadata)
+  * [Tables](#tables)
+* [Attribution](#attribution)
+
+
+## Cheatsheet
+
+### Cheatsheet: Content design, formatting, and language
+
+**[Contact information:](#contact-information)**
+* Use official Kubernetes contact information.
```

Comment:
> * Use official Kubernetes contact information.

What does it mean (clarification needed)?

Reply:
It's expanded on further down. The top portion is intended to just be  quick reference of the guidelines. The full block is this:
* Use official Kubernetes contact information.
   * Use official community contact email addresses. There should be no personal or work 
      contact information included in public documentation; instead use addresses like the SIG 
      Google groups or managed accounts such as community@kubernetes.io.
    * **Good example:** community@kubernetes.io
    * **Bad example:** bob@example.com